RESTfulAPI.md

```diff
@@ -56,6 +56,7 @@ Inside, the block, contain these fields:
 
   * name: can be ignored in **media**, in **contextual** instead specify the name of the topic addressed, and ale
 
+@vecna: Seems that this endpoint isn't implemented yet? [this request](http://213.108.108.94:8000/results/GBR) returns a 404.
 
```

Comment:
Sorry, I've not specify that the **legacy** are the current implemented, the specification of the new interface is just a specification (we have to start the 2.0 project to implement it)